fonts.css

```diff
@@ -0,0 +1,47 @@
+@font-face {
+    font-family: "Montserrat";
+    src: url("../fonts/montserrat-bold-webfont.woff2") format("woff2"),
+         url("../fonts/montserrat-bold-webfont.woff") format("woff"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/project_tesla/pullRequest?prid=5525151)